homeassistant/components/yalexs_ble/sensor.py

```diff
@@ -1,11 +1,23 @@
 """Support for yalexs ble sensors."""
 from __future__ import annotations
 
+from collections.abc import Callable
+from dataclasses import dataclass
+
 from yalexs_ble import ConnectionInfo, LockInfo, LockState
 
 from homeassistant import config_entries
-from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
-from homeassistant.const import SIGNAL_STRENGTH_DECIBELS_MILLIWATT
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    ELECTRIC_POTENTIAL_VOLT,
```

Comment:
ELECTRIC_POTENTIAL_VOLT is deprecated.

https://github.com/home-assistant/core/blob/9094d73df12c809f8ddbbd619c197ed79b1f7db0/homeassistant/const.py#L546

Reply:
Looks like I'm 5 days behind :)

#83287

Will adjust

Reply:
https://github.com/home-assistant/core/pull/83738